client/src/pages/EditProfile.js

```diff
@@ -41,12 +45,7 @@ function EditProfile() {
   const classes = useStyles();
 
   return (
-    <Box
-      display="flex"
-      flexWrap="nowrap"
-      style={{ paddingTop: "90px", minHeight: "100vh" }}
-      bgcolor="grey"
-    >
+    <Box display="flex" flexWrap="nowrap" bgcolor="grey">
```

Comment:
```suggestion
    <Box className={classes.boxContainer} display="flex" flexWrap="nowrap" bgcolor="grey">
```
It looks like you missed adding the class to this one